### ISSUE TITLE
[Table] Unable to build table in the latest versions

### ISSUE THREAD

**Issue by davidroeca** [fomantic/Fomantic-UI#942]
# Bug Report

Fomantic ui is unable to build from a fresh npm install

## Steps to reproduce
1. `npm init` a new directory
2. `npm i -D gulp fomantic-ui`
3. Follow all of the installation defaults
4. `npx gulp build --gulpfile semantic/gulpfile.js --cwd semantic/`

## Expected result

Successful build

## Actual result

A build that succeeds until it tries to build the table css:

```
[PluginError: Cannot create property '_index' on string '!important' in file /home/david/src/test/semantic/src/themes/default/collections/table.variables line no. 1] {
  message: "Cannot create property '_index' on string '!important' in file /home/david/src/test/semantic/src/themes/default/collections/table.variables line no. 1",
  stack: undefined,
  type: 'Syntax',
  filename: '/home/david/src/test/semantic/src/themes/default/collections/table.variables',
  index: NaN,
  line: 1,
  column: -1,
  callLine: NaN,
  callExtract: undefined,
  extract: [
    undefined,
    '/*******************************',
    '             Table'
  ],
  lineNumber: 1,
  fileName: '/home/david/src/test/semantic/src/themes/default/collections/table.variables',
  name: 'Error',
  plugin: 'gulp-less',
  showProperties: true,
  showStack: false,
  __safety: { toString: [Function: bound ] }
}
```

## Version
node `12.8.0` (also tested 10.16.2; same error)
npm `6.10.2`
fomantic-ui `2.7.7`
gulp `4.0.2`

---
EDIT: looks like someone else is encountering a similar error, it may be because the table.variables is the only file with `!important` https://stackoverflow.com/questions/57568988/gulp-less-fails-when-using-important

**Comment by davidroeca** [comment]
This issue is resolved if i `npm install -D gulp-less@3`. Not sure if this is ideal, but worth noting.

**Comment by davidroeca** [comment]
Looks like it's precisely what's happening here https://github.com/fomantic/Fomantic-UI/pull/939

**Comment by exoego** [comment]
It seems that upstream issue in `less.js`.
Ref: https://github.com/less/less.js/issues/3423

**Comment by lubber-de** [comment]
Fixed by less 3.10.3 in #950

**Comment by davidroeca** [comment]
This issue is fixed in fresh installs, so I'll to go ahead and close it
